### PR TITLE
feat (NSM-172) : Autocomplete component

### PR DESCRIPTION
**Pull request by radoslavdemirov**
- Added Autocomplete component by using the one found in @material-ui/lab

### CODE REVIEW

**Comment by BozhidarTodorov** [comment]
Билда май не минава. @radoslavdemirov , може ли да погледнеш?

**Comment by radoslavdemirov** [comment]
Мисля,че минава. Това е последния билд : 
https://jenkins.softwaregroup.com/job/ut/job/ut-front-react/job/PR-557/6/

image

На пул рекуеста също не пише за проблеми:

image

**Review by BozhidarTodorov — APPROVED** [review]

File: components/Autocomplete/index.js [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
-    dataSourceConfig: PropTypes.object,
-    onSelect: PropTypes.func,
-    searchText: PropTypes.string
+    options: PropTypes.array,
+    multiple: PropTypes.bool,
+    label: PropTypes.string,
+    placeholder: PropTypes.string,
+    selectedValue: PropTypes.object,
+    size: PropTypes.string,
+    inputValue: PropTypes.string
```
Comment (kalinkrustev) [inline components/Autocomplete/index.js:58]: I am not sure if it is safe to change the properties, which represent the public API of this component. I will need some time to check where it is used. Did you try to search in the many modules we have?

Comment (radoslavdemirov) [reply components/Autocomplete/index.js:58]: I searched and I didn't find any other places using this component

**Review by DimitarNeykovSG — APPROVED** [review]

**Review by stoberov — APPROVED** [review]

**Review by behrindzhambaz — APPROVED** [review]

**Pull request merged**